src/commonControls/numberControl/NumberControlAPL.ts

```diff
@@ -0,0 +1,167 @@
+                        },
+                    ],
+                },
+            },
+            mainTemplate: {
+                parameters: ['numPadData'],
+                items: {
+                    type: 'Container',
+                    width: '100vw',
+                    height: '100vh',
+                    direction: 'column',
+                    items: [
+                        {
+                            type: 'AlexaBackground',
+                        },
+                        {
+                            type: 'AlexaHeader',
+                            headerTitle: '${numPadData.headerTitle}',
+                            headerDivider: true,
+                            theme: 'dark',
+                        },
+                        {
+                            type: 'Container',
+                            height: '75%',
+                            alignItems: 'center',
+                            justifyContent: 'center',
+                            items: [
+                                {
+                                    type: 'EditText',
+                                    when: '${environment.aplVersion != null}',
+                                    id: 'editTextNumber',
+                                    style: 'EditStyle',
+                                    keyboardType: 'numberPad',
+                                    submitKeyType: 'go',
+                                    bind: [
+                                        {
+                                            name: 'NumberValue',
+                                            value: '${NumberValue}',
+                                            type: 'number',
+                                        },
+                                    ],
+                                    onSubmit: [
+                                        {
+                                            type: 'Sequential',
+                                            commands: [
+                                                {
+                                                    type: 'SendEvent',
+                                                    arguments: [
+                                                        '${numPadData.controlId}',
+                                                        '${event.source.value}',
+                                                    ],
+                                                },
+                                            ],
+                                        },
+                                    ],
+                                    accessibilityLabel: 'Enter a number',
+                                    width: '50vw',
+                                    validCharacters: '-0-9',
+                                    hint: 'Enter a number',
+                                    hintWeight: 'normal',
+                                    fontSize: '5vh',
```

Comment:
something for UX review.  The standard layouts use fixed pixel size fonts but I haven't researched the guidance -- should we treat all rectangle screens equivalently (using vw/vh for all sizing), or treat them as different sizes at same pixel density.  Certainly it is easier/simple to go with the first approach so maybe that is good.  We will need UX design input on this.

Reply:
Perhaps we should start a UX review wiki and collect questions of this kind.

Reply:
Adding this (and others) to UX questions. I did find that using vw/vh for font sizes worked best across screen sizes, but need to confirm what best practices are.